_episodes/02-setup.md

```diff
@@ -140,6 +140,23 @@ $ git config --list
 You can change your configuration as many times as you want: use the
 same commands to choose another editor or update your email address.
 
+> ## Default Branch Name
```

Comment:
I think this part needs to be merged with what has already been added as lines 108-128.